### ISSUE TITLE
create better admin logging tray for running docker container that is accesible outside the server

### ISSUE THREAD

**Issue by kaxada** [badging/BadgingAPI#17]

**Comment by FaithKovi** [comment]
hello @kaxada , can I get more information on this. I would like to work on it

**Comment by kaxada** [comment]
@FaithKovi there you go. let me know of anything that is not clear.

**Comment by FaithKovi** [comment]
@kaxada do you have a specific time duration for the log retention?

**Comment by FaithKovi** [comment]
@kaxada  Do you have a specific time duration for the log retention?

**Comment by kaxada** [comment]
No. The server is still huge can put it to anything more than a year. We can scale it down whenever. I hope you can make a draft PR in the coming week or so. @FaithKovi

**Comment by FaithKovi** [comment]
Okay that's fine then. You should have a draft PR soon @kaxada

**Comment by kaxada** [comment]
@FaithKovi you can use `http://localhost:{PORT}`. The `port` is whatever port you are running your local project on
